content/stack/release-notes/redisbloom/redisbloom-2.6-release-notes.md

```diff
@@ -12,11 +12,48 @@ aliases:
 ---
 ## Requirements
 
-RedisBloom v2.6.8 requires:
+RedisBloom v2.6.11 requires:
 
 - Minimum Redis compatibility version (database): 7.2
 - Minimum Redis Enterprise Software version (cluster): 7.2.4
 
+## v2.6.11 (January 2024)
+
+This is a maintenance release for RedisBloom 2.6
+
+Update urgency: `HIGH`: There is a critical bug that may affect a subset of users. Upgrade!
+
+Details:
+
+- Bug fixes:
+
+  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Potential crash on `CF.LOADCHUNK` (MOD-6344) - Additional fixes
```

Comment:
```suggestion
  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Potential crash on `CF.LOADCHUNK` (MOD-6344) 
  - Additional fixes
```

Reply:
No; this bug was partially fixed in the previous version. these are additional fixes to the same bug

Reply:
```suggestion
  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Additional fixes for potential crash on `CF.LOADCHUNK` (MOD-6344)
```